rpc/rpc-lib/src/protocol-common.h

```diff
@@ -309,9 +309,9 @@ enum glusterd_mgmt_v3_procnum {
     GLUSTERD_MGMT_V3_PRE_VALIDATE,
     GLUSTERD_MGMT_V3_BRICK_OP,
     GLUSTERD_MGMT_V3_COMMIT,
-    GLUSTERD_MGMT_V3_POST_COMMIT,
     GLUSTERD_MGMT_V3_POST_VALIDATE,
     GLUSTERD_MGMT_V3_UNLOCK,
+    GLUSTERD_MGMT_V3_POST_COMMIT,
     GLUSTERD_MGMT_V3_MAXVALUE,
```

Comment:
To avoid future mistakes, could you add a short and clear comment here saying that new entries **must** be added just above this value ?

Reply:
done